external/kafka/src/main/scala/org/apache/spark/streaming/kafka/KafkaCluster.scala

```diff
@@ -220,12 +226,22 @@ class KafkaCluster(val kafkaParams: Map[String, String]) extends Serializable {
   // https://cwiki.apache.org/confluence/display/KAFKA/A+Guide+To+The+Kafka+Protocol#AGuideToTheKafkaProtocol-OffsetCommit/FetchAPI
   // scalastyle:on
 
+  // this 0 here indicates api version, in this case the original ZK backed api.
+  def defaultConsumerApiVersion: Short = 0
```

Comment:
Shouldn't this be private? Do other parts of the spark/streaming call this function?